animation_nodes/data_structures/meshes/mesh_data.pyx

```diff
@@ -173,6 +191,13 @@ cdef class Mesh:
         for i in range(meshData.polygons.indices.length):
             self.polygons.indices.data[polygonIndicesOffset + i] += vertexOffset
 
+    def getVertexColorLayer(self, str colorLayerName):
+        cdef long i    
+        for i in range(len(self.getVertexColorLayers())):
+            if self.getVertexColorLayers()[i][0] == colorLayerName:
+                return self.getVertexColorLayers()[i]    
+            else:
+                return None
```

Comment:
`vertexColorLayers` is an ordered dictionary, you can simply get the vertex color layer using subscript notation. Also, move this function below the `getVertexColorLayerNames`.

Reply:
Is this correct way,
```
def getVertexColors(self, str colorLayerName):
        return self.vertexColorLayers.get(colorLayerName, None)
```
This function return the colors corresponds to the colorLayerName.

Reply:
Yes, looks good.

Reply:
I have added this function.